diff --git a/node_modules/element-plus/es/components/tabs/src/tab-nav.mjs b/node_modules/element-plus/es/components/tabs/src/tab-nav.mjs
--- a/node_modules/element-plus/es/components/tabs/src/tab-nav.mjs
+++ b/node_modules/element-plus/es/components/tabs/src/tab-nav.mjs
@@ -286,8 +286,8 @@ const TabNav = /* @__PURE__ */ defineComponent({
 				"style": navStyle.value,
 				"role": "tablist",
 				"onKeydown": changeTab,
-				"onWheel": handleWheel,
-				"onTouchstart": handleTouchStart,
-				"onTouchmove": handleTouchMove,
-				"onTouchend": handleTouchEnd,
-				"onTouchcancel": handleTouchEnd
+				"onWheel": scrollable.value ? handleWheel : void 0,
+				"onTouchstart": scrollable.value ? handleTouchStart : void 0,
+				"onTouchmove": scrollable.value ? handleTouchMove : void 0,
+				"onTouchend": scrollable.value ? handleTouchEnd : void 0,
+				"onTouchcancel": scrollable.value ? handleTouchEnd : void 0
diff --git a/node_modules/element-plus/lib/components/tabs/src/tab-nav.js b/node_modules/element-plus/lib/components/tabs/src/tab-nav.js
--- a/node_modules/element-plus/lib/components/tabs/src/tab-nav.js
+++ b/node_modules/element-plus/lib/components/tabs/src/tab-nav.js
@@ -287,8 +287,8 @@ const TabNav = /* @__PURE__ */ vue.defineComponent({
 				"style": navStyle.value,
 				"role": "tablist",
 				"onKeydown": changeTab,
-				"onWheel": handleWheel,
-				"onTouchstart": handleTouchStart,
-				"onTouchmove": handleTouchMove,
-				"onTouchend": handleTouchEnd,
-				"onTouchcancel": handleTouchEnd
+				"onWheel": scrollable.value ? handleWheel : void 0,
+				"onTouchstart": scrollable.value ? handleTouchStart : void 0,
+				"onTouchmove": scrollable.value ? handleTouchMove : void 0,
+				"onTouchend": scrollable.value ? handleTouchEnd : void 0,
+				"onTouchcancel": scrollable.value ? handleTouchEnd : void 0
